qiskit_experiments/quantum_volume/qv_experiment.py

```diff
@@ -0,0 +1,261 @@
+            self._previous_trials = int(len(experiment_data.data()) / 2)
+
+        # Filter kwargs
+        run_options = self.__run_defaults__.copy()
+        circuit_options = {}
+        for key, value in kwargs.items():
+            if key in _TRANSPILE_OPTIONS or key in self._circuit_options:
+                circuit_options[key] = value
+            else:
+                run_options[key] = value
+
+        # Generate and run circuits
+        transpiled_circuits, circuits = self.transpiled_circuits(backend, **circuit_options)
+        for circ in transpiled_circuits:
+            circ.metadata["is_simulation"] = False
+            circ.measure_active()
+        qobj = assemble(transpiled_circuits, backend, **run_options)
+        job = backend.run(qobj)
+
+        sim_data = self._get_ideal_data(circuits, run_options)
+        # Add Jobs to ExperimentData
+        experiment_data.add_data([job, sim_data])
+
+        # use 'return_figures' parameter if given
+        return_figures = kwargs.get("return_figures", False)
+        # Queue analysis of data for when job is finished
+        if self.__analysis_class__ is not None:
+            if return_figures:
+                # pylint: disable = not-callable
+                _, figures = self.__analysis_class__().run(experiment_data, **kwargs)
+            else:
+                self.__analysis_class__().run(experiment_data, **kwargs)
+
+        # Return the ExperimentData future
+        if return_figures:
+            return experiment_data, figures
+        return experiment_data
+
+    def add_trials(self, additional_trials):
+        """
+        Add more trials to the experiment
+        Args:
+            additional_trials (int): The amount of trials to add
+        """
+        self._trials += additional_trials
+
+    @property
+    def trials(self):
+        """Return number of trials in the experiment"""
+        return self._trials
+
+    def _get_ideal_data(self, circuits, run_options):
+        """
+        in case the user do not have aer installed - use Terra to calculate the ideal state
+        Args:
+            circuits: the circuits to extract the ideal data from
+        Returns:
+            dict: data object with the circuit's metadata
+                  and the probability for each state in each circuit
+        """
+        if self._simulation_backend:
+            for circuit in circuits:
+                circuit.metadata["is_simulation"] = True
+                circuit.save_probabilities()
+            return execute(circuits, backend=self._simulation_backend, **run_options)
+        else:
+            from qiskit.quantum_info import Statevector
+            import numpy as np
+
+            sim_obj = []
+            for circuit in circuits:
+                circuit.metadata["is_simulation"] = True
+                state_vector = Statevector(circuit)
+                prob_vector = np.multiply(state_vector, state_vector.conjugate())
```

Comment:
why can't you use the `probabilities` or `probabilities_dict` method of the `Statevector` class?

Reply:
I didn't know about them. changed :)